test/integration/protocol_integration_test.cc

```diff
@@ -425,13 +442,13 @@ TEST_P(DownstreamProtocolIntegrationTest, MissingHeadersLocalReply) {
 
 // Regression test for https://github.com/envoyproxy/envoy/issues/10270
 TEST_P(ProtocolIntegrationTest, LongHeaderValueWithSpaces) {
+  EXCLUDE_DOWNSTREAM_HTTP3
```

Comment:
some of the header tests should be fixed by 15381 but others need configurable header size to pass. We'll have to figure that one out at some point :-)